tracker/event_tracker.go

```diff
@@ -36,6 +37,7 @@ func NewEventTracker(
 	numBlockConfirmations uint64,
 	startBlock uint64,
 	logger hcf.Logger,
+	pollInterval common.Duration,
```

Comment:
Why you haven't put `time.Duration`? `common.Duration` is there solely for JSON (de)serialization
```suggestion
	pollInterval time.Duration,
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1887/commits/f47c9ae2d3128f5fa8fa5bd00b49ef665ae38f18